installer/release.sh

```diff
@@ -326,12 +326,14 @@ test -z "$arm64_artifacts_directory" || {
 die "Could not include ARM64 artifacts"
 
 etc_gitconfig_dir="${etc_gitconfig%/gitconfig}"
+etc_sshconfig_dir="${${etc_gitconfig%/ssh%/ssh_config}"
```

Comment:
I'm pretty sure this doesn't work. The double `${` with a single `}` cant be correct. I also don't think you can just concatenate these `%` patterns. And even if both of that would work, what are you trying to do? Strip a path that ends in `/etc/gitconfig` of a trailing `/ssh/ssh_config`?